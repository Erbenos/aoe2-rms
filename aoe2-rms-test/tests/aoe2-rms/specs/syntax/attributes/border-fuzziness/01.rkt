#lang aoe2-rms
; Central desert with very fuzzy borders

<LAND-GENERATION>
(create-land
 (terrain-type 'DESERT)
 (land-position 50 50)
 (land-percent 100)
 (left-border 40)
 (right-border 40)
 (top-border 40)
 (bottom-border 40)
 (border-fuzziness 2))
