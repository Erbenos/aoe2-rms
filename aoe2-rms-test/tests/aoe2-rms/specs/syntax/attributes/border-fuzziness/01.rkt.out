<LAND_GENERATION>
create_land {
  terrain_type DESERT
  land_position 50 50
  land_percent 100
  left_border 40
  right_border 40
  top_border 40
  bottom_border 40
  border_fuzziness 2
}
